pom.xml

```diff
@@ -152,7 +152,7 @@ flexible messaging model and an intuitive client API.</description>
     <mysql-jdbc.version>8.0.11</mysql-jdbc.version>
     <postgresql-jdbc.version>42.2.25</postgresql-jdbc.version>
     <clickhouse-jdbc.version>0.3.2</clickhouse-jdbc.version>
-    <mariadb-jdbc.version>2.6.0</mariadb-jdbc.version>
+    <mariadb-jdbc.version>3.0.3</mariadb-jdbc.version>
```

Comment:
MariaDB Connector/J 3.0.3 is NOT fully compatible with the latest release of version 2.7.

Reply:
@nodece please elaborate more about the incompatibility.

Reply:
The information from https://mariadb.com/kb/en/mariadb-connector-j-303-release-notes/.
> NOTE: MariaDB Connector/J 3.0.3 is NOT fully compatible with the latest release of version 2.7.